### PR TITLE
fix(curriculum): Changed Mongoose version to avoid warnings in boilerplate's Node console when working on Mongoose challenges

### PR DESCRIPTION
**Pull request by RandellDawson**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

As mentioned [in this PR's comments](https://github.com/freeCodeCamp/boilerplate-mongomongoose/pull/36#issuecomment-1202987911), I found simply removing `mongodb` and changing `mongoose` version to `^5.11.15` gets rid of all the warning messages in the Node console and does not affect any of the challenges in the MongoDB and Mongoose section challenges.  See [related boilerplate PR](https://github.com/freeCodeCamp/boilerplate-mongomongoose/pull/48) where I change the `package.json` to remove `mongodb` and change the `mongoose` version to `^5.11.15`.

This PR updates the instructions and tests to work with the changes in the corresponding boilerplate PR (mentioned above).

### CODE REVIEW

**Review by naomi-lgbt — DISMISSED** [review]
This is a good idea.

I wonder if we should update the block description:
> Mongoose is a popular npm package that is often installed alongside Mongo.

**Comment by RandellDawson** [comment]
> This is a good idea.
> 
> I wonder if we should update the block description:
> 
> > Mongoose is a popular npm package that is often installed alongside Mongo.

Probably.  What did you have in mind?

**Comment by RandellDawson** [comment]
**Update:**  I have confirmed using a replit, that installing `mongoose@5.11.15` without also installing `mongodb` as a dependency does not affect the project.  I was able to use my same code that passed with both `mongodb@~3.6.0` and `mongoose@~5.4.0` with no warning messages and passed all the challenges.

**Comment by naomi-lgbt** [comment]
> Probably. What did you have in mind?

Maybe something like "Mongoose is a popular npm package for interacting with MongoDB"?

> we could rename the section to just Mongoose

I think the course name is okay - we are teaching campers about MongoDB. It's just the mention of the `mongo` package that needs to be removed.

**Comment by RandellDawson** [comment]
> Maybe something like "Mongoose is a popular npm package for interacting with MongoDB"?

I like it.

**Comment by naomi-lgbt** [comment]
Sure I can do that!

File: curriculum/challenges/english/05-back-end-development-and-apis/mongodb-and-mongoose/install-and-set-up-mongoose.md [hunk 1]
```diff
@@ -64,8 +44,8 @@ mongoose.connect(<Your URI>, { useNewUrlParser: true, useUnifiedTopology: true }
       assert.property(packJson.dependencies, 'mongoose');
       assert.match(
         packJson.dependencies.mongoose,
-        /^\~5\.4\.0/,
-        'Wrong version of "mongoose". It should be ~5.4.0'
+        /^\^5\.11\.15/,
+        'Wrong version of "mongoose". It should be ~5.11.15'
```
Comment (scissorsneedfoodtoo) [inline curriculum/challenges/english/05-back-end-development-and-apis/mongodb-and-mongoose/install-and-set-up-mongoose.md:48]: Just noticed one character here that can be updated:

```suggestion
        'Wrong version of "mongoose". It should be ^5.11.15'
```

**Review by scissorsneedfoodtoo — CHANGES_REQUESTED** [review]

**Review by scissorsneedfoodtoo — APPROVED** [review]

**Review by Sboonny — APPROVED** [review]

**Comment by Sboonny** [comment]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Pull request merged**